miio/airhumidifier.py

```diff
@@ -180,6 +183,11 @@ def motor_speed(self) -> Optional[int]:
     @property
     def depth(self) -> Optional[int]:
         """The remaining amount of water in percent."""
+
+        # MODEL_HUMIDIFIER_CB2 127 without water tank. 125 = 100% water
+        if self.device_info.model == MODEL_HUMIDIFIER_CB2:
```

Comment:
You could introduce a new boolean property to indicate the water tank is detached.

Reply:
Hi @syssi - do you want to get this change already into this PR, or shall we merge this already for the next release?